spec/controllers/teams_controller_spec.rb

```diff
@@ -6,31 +6,6 @@
   let(:team) { FactoryBot.create :team }
 
   describe 'GET #edit' do
-    context 'when signed in as a team leader without rights to manage all teams' do
```

Comment:
This test is removed because the page is now only for editing banned competitors and the test for that already exists below.

Alternatively creating a similar unit test for user_roles will come up once the migration is completed. Currently since the user_roles_controller is in an intermediate stage, it might be difficult to have unit tests there now.